apps/bundle_deploy/build_model.py

```diff
@@ -18,12 +18,15 @@
 
 import argparse
 import os
+import pathlib
+
 from tvm import relay
 import tvm
 from tvm import runtime as tvm_runtime
 import logging
 from tvm.relay.backend import Runtime
 from tvm.contrib import cc as _cc
+from tvm.micro import copy_crt_config_header
```

Comment:
does this add a dependency on USE_MICRO being enabled that wasn't here previously, should this be conditional on that?